### DISCUSSION TITLE
Collaborator 是不是和 Member 一样需要手动同意才能加入？

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#3661] [提问] [answered]
有没有什么办法自动通过（至少现阶段我需要这个功能，因为我同时 Watch 了 PCL 和 PCL-1930 两个仓库邮件高达 819 封，略过了这个邀请
![image](https://github.com/Hex-Dragon/PCL2/assets/160083863/cba78dfe-8eac-4410-9a95-c5c9f3258a8d)
2/22 发给我的我到 4/08 才看到

**Comment by MoYuan-CN** [comment, marked as answer]
不行的好像

**Comment by Deep-Dark-Forest** [comment]
Collaborator 没法自动通过，因为你也得看人家愿不愿意…
另外，PCL **不会**……

**Comment by HaoTian22** [comment]
另外要根本解决问题，建议使用邮件分类（Tag/Move/...），Outlook在自动化处理邮件规则方面我觉得是一流的

**Reply by allMagicNB** [reply]
~~其实根本地解决问题取消 Watch 就得（~~

**Reply by wuliaodexiaoluo** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/160083863/107af8d3-52cb-44f9-a76a-d7e458cda702)
主要是如果不 Watch 下次发 Collaborate 就轮不到我了(虽然可有可无就是了，但有时候 Label 打的真的血压上来了

**Reply by allMagicNB** [reply]
> ![image](https://github.com/Hex-Dragon/PCL2/assets/160083863/107af8d3-52cb-44f9-a76a-d7e458cda702)
> 主要是如果不 Watch 下次发 Collaborate 就轮不到我了(虽然可有可无就是了，但有时候 Label 打的真的血压上来了

不 Watch 也能收到 Invitation 的，这个不取决于你的 Watch 所有活动……

**Reply by wuliaodexiaoluo** [reply]
龙猫会看活跃度的，不 Watch 长期潜水也不可能 Invitation 吧
